### PR TITLE
Feature/travis release

### LINKED ISSUE #69: Publish a new version every time something merges to master
in Travis CI.

Tasks to be done:

Every time something merges to master with the pattern [fix] create a minor version (Bump third digit)
Every time something merges to master with the pattern [feature] create a new version (Bump second digit)
Every time something merges to master with the pattern [breaking-change] create a new major version (Bump first digit)
Every time something merges to master with the pattern [docs] rebuild just the docs and publish them

Build the new changes
Build the docs
Publish the docs
Publish the changes to pypy

Update contributing.md to ask for those specific tags on PR titles.

### PR DESCRIPTION
**Pull request by brunomichetti**
closes #69 

- Now all the changes are automatically uploaded to Pypi.
- CONTRIBUTING is updated to explain how to bump the version.
- TODO:
Automatically release updated documentation.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,6 +69,16 @@ x = 3
 print('Number in var x is: {x}'.format(x=x))
 ```
 
+## Update the version
+
+Before creating the pull request, please update the version in `drip/__inti__.py` following the next rules:
+
+- If you are doing a fix or a very small feature, then bump the third digit.
+- If you are adding a new feature, then bump the second digit.
+- If you are doing a breaking changes or a very big feature, then bump the first digit.
```
Comment (nsantos16) [inline CONTRIBUTING.md:78]: I think we need to define a better way to establish the version releases(in my opinion semantic versioning fits for this project).
Here's a couple of references:
- [Maintaining Open Source Projects: Versioning and Releasing](https://thoughtbot.com/blog/maintaining-open-source-projects-versioning#time-based-releases)
- [Semantic versioning](https://semver.org/)
@rootstrap/python-maintainers

**Review by kaozdl — DISMISSED** [review]

**Review by kaozdl — APPROVED** [review]

**Review by kaozdl — CHANGES_REQUESTED** [review]
Shouldn't this be saved somewhere?

**Comment by brunomichetti** [comment]
> Shouldn't this be saved somewhere?

What do you mean?

**Review by kaozdl — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -60,3 +60,14 @@ jobs:
         - coverage run manage.py test
         - coverage xml
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+    - stage: Pypi
+      # Do it only when merge on master branch
+      if: (branch = master) AND (commit_message =~ /^Merge pull request.*/)
+      script: skip
+      deploy:
+        provider: pypi
+        username: brunomichetti
```
Comment (jiss2891) [inline .travis.yml:69]: Do we really want to keep this info on the repo?

**Review by jiss2891 — COMMENTED** [review]
Left a comment

File: .travis.yml (hunk 2 above)
Comment (jiss2891) [reply .travis.yml:69]: we should manage this through environment variables.

**Review by kaozdl — APPROVED** [review]

**Review by jiss2891 — APPROVED** [review]

**Pull request merged**